### PR TITLE
docs: Document importing / exporting Identity (Kratos) configs through the CLI

### PR DESCRIPTION
**Pull request by tomekpapiernik**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

## Further comments

Added documentation on importing/exporting configuration which currently applies to Identity / Kratos. As such this is labeled `Configure Identity Service`.

Additionally, updated the existing `CLI basics` document as it didn't contain information about creating projects through the CLI.

### CODE REVIEW

File: docs/guides/cli/15_config-identity-service.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### Export to File
+
+For a convenient way to get the Identity Service configuration 
+working in self-hosted Ory Kratos, save the configuration to a YAML file.
+
+Run this command:
+
+```bash
+ory get identity-config <project-id> --format yaml >identity-config.yaml
```
Comment (piotrmsc) [inline docs/guides/cli/15_config-identity-service.mdx:49]:
```suggestion
ory get identity-config <project-id> --format yaml > identity-config.yaml 
```

File: docs/guides/cli/15_config-identity-service.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Export Configuration from Ory Cloud
+
+To get your project's Identity Service configuration, run this command:
+
+:::info
+
+You must be signed in to an Ory Cloud account to perform this action. 
+Read [this document](./cli-basics#use-cli-with-existing-cloud-account) to learn more. 
+
```
Comment (piotrmsc) [inline docs/guides/cli/15_config-identity-service.mdx:28]: would be nice to add a tip with listing all available projects

File: docs/guides/cli/15_config-identity-service.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+working in self-hosted Ory Kratos, save the configuration to a YAML file.
+
+Run this command:
+
+```bash
+ory get identity-config <project-id> --format yaml >identity-config.yaml 
+```
+
+## Import Configuration to Ory Cloud
+
```
Comment (piotrmsc) [inline docs/guides/cli/15_config-identity-service.mdx:53]: it would be nice to add a hint with creating a project as an entry to this page itself

**Review by piotrmsc — COMMENTED** [review]
Thank you for the update :) I've left some minor comments

File: docs/guides/cli/15_config-identity-service.mdx (hunk 3 above)
Comment (tomekpapiernik) [reply docs/guides/cli/15_config-identity-service.mdx:61]: I added one that links to the CLI docs and the console. I didn't put in a "tip" or "note" frame though as there are already a few of those in the document, especially in its first part.

**Review by piotrmsc — APPROVED** [review]

**Pull request merged**